Test/Test/ConsoleReader.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+#include <conio.h>
+#include <string>
+
+#include "ConsoleReader.h"
+
+#include <log4cpp/Portability.hh>
```

Comment:
Do we really need all these includes?

Reply:
I guess not, it seems that I can throw away most of them